### DISCUSSION TITLE
How do i overwrite default keybinds?

### DISCUSSION THREAD

**Discussion by kamalthapalia** [prasanthrangan/hyprdots#1373] [Q&A]
I want app launcher to open on super key press instead of super + a, and want to modify other keybinds too.
Thanks

**Comment by dieBakterie** [comment]
In the `keybindings.conf`

**Reply by kamalthapalia** [reply]
bind = $mainMod,  ,exec, pkill -x rofi || $scriptsPath/rofilaunch.sh d 
this doesnt work 
how do i make super key only??
also i want different looks on launcher where do i change that?

**Reply by se7en-x230** [reply]
you can't make super only otherwise everytime you press another combination it will also launch rofilauncher

bind = $mainMod, (YOUR KEY) ,exec ...
make sure the key is not in use or unbind it first

**Reply by kRHYME7** [reply]
@kamalthapalia  

` bindr = SUPER, SUPER_L, exec, pkill -x rofi || $scrPath/rofilaunch.sh d # launch desktop applications `

https://wiki.hyprland.org/Configuring/Binds/

**Comment by se7en-x230** [comment]
I would "unbind" and "bind" in userprefs.conf

![image](https://github.com/prasanthrangan/hyprdots/assets/69777713/6ec19e58-347f-4a5c-8d83-327c2d5df45e)
